pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -137,7 +137,7 @@ var (
 			"ISTIOD_RA_ISTIO_API").Get()
 
 	// TODO: Likely to be removed and added to mesh config
-	k8sSigner = env.RegisterStringVar("K8S_SIGNER", "",
+	k8sSigner = env.RegisterStringVar("K8S_SIGNER", constants.DefaultK8SSigner,
```

Comment:
A more detailed comment will help users to configure this env variable. For example, besides constants.DefaultK8SSigner, which other signers are supported? Are all signers on https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/ supported?

Reply:
Right now, NONE of the in-built K8s signers can be used (based on #22161). This PR is to provide support in the future for (a) getting Istiod's certificate signed by an External CA that servers as a K8s CSR signer or (b) a possible in-built signer that Kubernetes provides that satisfies Istiod's requirements. The exact details of the external CA and signer are out of scope of this PR. The nature of the certificate that they need to sign is documented here. [https://docs.google.com/document/d/1kvmZgJ-QZByMlhqZDTdoxwHtJQxsrl5kj2-4HB4ip80/edit?resourcekey=0-OcIivFMxKkGxRB5KRPcGvg#heading=h.qlmnvhzb2ib2](url)